packages/ui/src/FormGroupHeading.tsx

```diff
@@ -24,14 +32,17 @@ export const FormGroupHeading: React.FC<Props> = ({
       p={2}
       borderRadius={8}
       w='100%'
-      onClick={onClick}
+      tabIndex={1}
+      cursor='pointer'
```

Comment:
Making the cursor a pointer and giving it a tabIndex across the entire header may give a user the mistaken understanding that clicking this will open / collapse the header, but only clicking the button inside will.

Reply:
I think if it's collapsed having it be "click anywhere" is preferable